### ISSUE TITLE
[Bug] [Code: PRAxMMMV] Unable to translate Portuguese (Portugal)

### ISSUE THREAD

**Issue by ConfusedVorlon** [FlineDev/ReMafoX#69]
## Steps to Reproduce:

using Microsoft Translator:
1. Add a localisation file in Portuguese (Portugal)
2. iOS seems to automatically copy my English translations
3. delete one key so that Remafox has something to translate

## Expected Behavior:

it translates

## Current Behavior:

it fails...

> Microsoft Translator request failed with error: The request was sent and the server responded with a body, but the conversion of the body to the given type failed. Type: Array<MTTranslateResponseEntry>. System Info: The data couldn’t be read because it isn’t in the correct format. (Error Code: APIxV)

## Environment

PRAxMMMV

<details>
<summary>Show environment details</summary>

- App Version: 1.2.1 (Build 57)
- System Version: 13.0 (Build 22A380)
- System CPU: Apple M1 Ultra
- Tier: Free

</details>

**Comment by ConfusedVorlon** [comment]
note - the same process for other languages works without issue

**Comment by ConfusedVorlon** [comment]
wild guess - could this be a case sensitivity issue?

iOS uses pt-PT for the language code.
MSTranslator seems to use pt-pt

**Comment by Jeehut** [comment]
@ConfusedVorlon Thank you for reporting this issue. Let me check it right away ...

**Comment by Jeehut** [comment]
Alright, I found the issue:

I'm making a request to `https://api.cognitive.microsofttranslator.com/translate?to=pt-pt&from=en&api-version=3.0` (note the `pt-pt` param value I'm sending) as documented on their API [here](https://learn.microsoft.com/en-us/azure/cognitive-services/translator/language-support). The translation itself works. But the response body I get is:
```JSON
[{"translations":[{"text":"<translated text>","to":"pt-PT"}]}]
```
Note the `pt-PT` code in the response. When parsing this back to a `Codable` in RemafoX, it seems things break. It appears that their documentation isn't quite right, sending `pt-PT` instead get things working again. I just made the adjustment. The fix will be part of the next patch release `1.2.2` which I'll upload very soon.

Thanks again for reporting – your guess was pretty good, too! I appreciate it. 🙌
